network/ddclient-configuration.sh

```diff
@@ -23,17 +23,26 @@ You have to setup an account before you can start.\n\nPlease choose your DynDNS-
 $MENU_GUIDE\n\n$RUN_LATER_GUIDE
 If your DDNS provider isn't already supported, please open a new issue here:\n$ISSUES" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 "Cloudflare" "(cloudflare.com)" \
+"deSEC" "(desec.io)" \
 "Strato" "(strato.de)" 3>&1 1>&2 2>&3)
 
 case "$choice" in
     "Cloudflare")
         PROVIDER="Cloudflare"
-        INSTRUCTIONS="Register an email address for your domain and get an Cloudflare API-key"
+        INSTRUCTIONS="register an email address for your domain and get an Cloudflare API-key"
         GUIDE="https://www.techandme.se/setup-multiple-accounts-with-ddclient-and-cloudflare/"
         PROTOCOL="cloudflare"
         SERVER="www.cloudflare.com"
         USE_SSL="yes"
     ;;
+    "deSEC")
+        PROVIDER="deSEC"
+        INSTRUCTIONS="get a DDNS account with password"
+        GUIDE="https://www.os-privacy.de/dyndns-desec-3181/"
```

Comment:
Please ad this instead: https://desec.io/#!

Reply:
Or maybe we should actually create the account? https://desec.readthedocs.io/en/latest/auth/account.html#register-account